### PR TITLE
Update BasicInsightRequest.java

### PR DESCRIPTION
**Pull request by poornapragna877**
Removed unnecessary references to "this" keyword

_Describe your changes here_

## Contribution Checklist
* [ ] Unit tests!
* [ ] Updated [CHANGELOG.md](CHANGELOG.md)
* [ ] My name is in [CONTRIBUTORS.md](CONTRIBUTORS.md)

### CODE REVIEW

File: src/main/java/com/vonage/client/insight/BasicInsightRequest.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public static Builder builder(String number) {
          * @param number A single phone number that you need insight about in national or international format.
          */
         public Builder(String number) {
-            this.number = number;
```
Comment (yallen011) [inline src/main/java/com/vonage/client/insight/BasicInsightRequest.java:59]: The `this` keyword was only supposed to be removed from places that didn't need it. You removed the keyword from places that did need it as well as from places that did need it which are causing the test to fail. Can you fix the changes that are causing the tests to fail. Then run a `./gradlew build`(on a mac) or `gradlew.bat build`(on a Windows machine) on your local machine to make sure the build pass then push up the changes. If you need help understanding the changes that need to be made please let me know.

**Review by yallen011 — CHANGES_REQUESTED** [review]

**Comment by yallen011** [comment]
please add the changes from your closed PR #313 to this one. Thank you

**Comment by lukeocodes** [comment]
@poornapragna877 please address the requested changes so we can merge or close this PR. Thanks for your contribution!

**Comment by yallen011** [comment]
This PR has gone stale. No response from requestor. Closing

**Pull request closed without merging**